src/hidimstat/desparsified_lasso.py

```diff
@@ -152,271 +121,378 @@ def desparsified_lasso(
-        Data.
+    beta_hat : ndarray, shape (n_features, n_times)
+        Estimated parameter matrix from desparsified group Lasso.
 
-    Y : ndarray, shape (n_samples, n_times)
-        Target.
+    theta_hat : ndarray, shape (n_times, n_times)
+        Estimated precision matrix (inverse covariance).
 
-    cov : ndarray, shape (n_times, n_times), optional (default=None)
-        If None, a temporal covariance matrix of the noise is estimated.
-        Otherwise, `cov` is the temporal covariance matrix of the noise.
+    omega_diag : ndarray, shape (n_features,)
+        Diagonal elements of the precision matrix.
 
-    test : str, optional (default='chi2')
-        Statistical test used to compute p-values. 'chi2' corresponds
-        to a chi-squared test and 'F' corresponds to an F-test.
+    test : {'chi2', 'F'}, default='chi2'
+        Statistical test for computing p-values:
+        - 'chi2': Chi-squared test (recommended for large samples)
+        - 'F': F-test (better for small samples)
 
-    max_iter : int, optional (default=5000)
-        The maximum number of iterations when regressing, by Lasso,
-        each column of the design matrix against the others.
+    Returns
+    -------
+    pval : ndarray, shape (n_features,)
+        Raw p-values, numerically accurate for positive effects
+        (p-values close to 0).
 
-    tol : float, optional (default=1e-3)
-        The tolerance for the optimization of the Lasso problems: if the
-        updates are smaller than `tol`, the optimization code checks the
-        dual gap for optimality and continues until it is smaller than `tol`.
+    pval_corr : ndarray, shape (n_features,)
+        P-values corrected for multiple testing using
+        Benjamini-Hochberg procedure.
 
-    residual_method : str, optional (default='lasso')
-        Method used for computing the residuals of the Nodewise Lasso.
-        Currently the only method available is 'lasso'.
+    one_minus_pval : ndarray, shape (n_features,)
+        1 - p-values, numerically accurate for negative effects
+        (p-values close to 1).
 
-    alpha_max_fraction : float, optional (default=0.01)
-        Only used if method='lasso'.
-        Then alpha = alpha_max_fraction * alpha_max.
+    one_minus_pval_corr : ndarray, shape (n_features,)
+        1 - corrected p-values.
 
-    noise_method : str, optional (default='simple')
-        If 'simple', the correlation matrix is estimated by taking the
-        median of the correlation between two consecutive time steps
-        and the noise standard deviation for each time step is estimated
-        by taking the median of the standard deviations for every time step.
-        If 'AR', the order of the AR model is given by `order` and
-        Yule-Walker method is used to estimate the covariance matrix.
+    Notes
+    -----
+    The chi-squared test assumes asymptotic normality while the F-test
+    makes no such assumption and is preferable for small sample sizes.
```

Comment:
The F test also assumes asymptotic normality.

Reply:
See the PR https://github.com/mind-inria/hidimstat/pull/127 for this problem.

Reply:
This part is not touched in #127 ?

Reply:
I just report your remark in the PR #127, I didn't have the time fix it for the moment.